AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1407,6 +1407,7 @@ cointelegraph.com##.subscription-form
 ramsaydiagnostics.ru##.mailing-box
 inmyroom.ru##.c-subscribe-form
 wakacje.pl##footer[data-area="footer"] > form[action="/newsletter.html"]
+cryptopotato.com##.widget_mo_optin_widgets
```

Comment:
redundant by `cryptopotato.com##.moOptinForm`

Reply:
If we remove this rule we leave a leftover "JOIN OUR NEWSLETTER"